snuba/clickhouse/http.py

```diff
@@ -39,12 +41,31 @@ def __default(self, value: Any) -> Any:
         else:
             raise TypeError
 
-    def encode(self, value: WriterTableRow) -> JSONRow:
+    def encode(self, value: WriterTableRow) -> bytes:
         return cast(
             bytes, rapidjson.dumps(value, default=self.__default).encode("utf-8")
         )
 
 
+class ValuesRowEncoder(Encoder[bytes, WriterTableRow]):
```

Comment:
PLease add some unit tests for this before merging.

Reply:
I've added a few unit tests. Let me know if you'd like coverage of anything else.

Reply:
It is ok to me.
Optional:
I would add some more complex expression(like function containing a function). 
Also what happens if you define an alias in an expression in the INSERT? I never tried.